topic-controller/src/main/java/io/strimzi/controller/topic/TopicSerialization.java

```diff
@@ -18,15 +18,20 @@
 package io.strimzi.controller.topic;
 
 import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonParseException;
```

Comment:
Do we still use YAML somwehere? If not, do we need Jakson library? Can't we do with the JSON support from Vert.x? We will need to productize this as well and every library would mean more work.

Reply:
Of course we can change it, but it's actually a dependency of Vert.x, so I don't think it gains you anything.

Reply:
Ok, you are right. If it is also Vert.x dependency than it doesn't matter.

Reply:
I think that we should have the same approach (using the same classes). The cluster controller is relying on the Vert.x JSON classes. I know that components developed from different people could have huge differences even when they live in the same repo but having a minimum of "consistency" looks better to me. It doesn't mean that you have to change to use Vert.x JSON classes, maybe we should change the cluster controller to use Jakson. Or just leaving as it is ... maybe I'm just mumbling :-)